streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestDriver.java

```diff
@@ -732,4 +822,52 @@ private static List<TopicPartition> getAllPartitions(final KafkaConsumer<?, ?> c
         return partitions;
     }
 
+    private static void verifyAllTransactionFinished(final KafkaConsumer<byte[], byte[]> consumer,
```

Comment:
Seems we copied this from `EosTestDriver`? Why did we drop the repartition step?

If we use repartitioning in the EOS test, but not in the Smoke test, we should add a partition step to the smoke test, to not reduce test coverage.